pkg/omnimatcher/omnimatcher.go

```diff
@@ -42,11 +44,12 @@ func New(m []driver.Matcher) OmniMatcher {
 }
 
 // Vulnerable will call each Matcher's Vulnerable method until one returns true.
-func (om OmniMatcher) Vulnerable(record *claircore.IndexRecord, vuln *claircore.Vulnerability) bool {
+func (om OmniMatcher) Vulnerable(ctx context.Context, record *claircore.IndexRecord, vuln *claircore.Vulnerability) (bool, error) {
 	for _, m := range om {
-		if m.Vulnerable(record, vuln) {
-			return true
+		match, _ := m.Vulnerable(ctx, record, vuln)
```

Comment:
probably should not ignore this error, 

however this ties into my other comments of making sure rhel matcher handles not having content set mapping file in memory without erroring.

Reply:
Fixed